### PR TITLE
Hypertargets for Omega, block-reward-R-b, Gamma, Phi and Pi

### PR DESCRIPTION
**Pull request by jamesray1**

### CODE REVIEW

File: Paper.tex [hunk 1]
```diff
@@ -1107,14 +1107,14 @@ \subsection{Reward Application}
 R & \equiv & (1 + \frac{1}{8} (U_i - {B_H}_i)) R_b
 \end{eqnarray}
 
-If there are collisions of the beneficiary addresses between ommers and the block (i.e. two ommers with the same beneficiary address or an ommer with the same beneficiary address as the present block), additions are applied cumulatively.
+If there are collisions of the beneficiary addresses between ommers and the block (i.e. two ommers with the same beneficiary address or an ommer with the same beneficiary address as the present block), additions are applied cumulatively.\hypertarget{block-reward-R-b}{block-reward-R-b}
```
Comment (pirapira) [inline Paper.tex]: Why isn't the target on the next equation?

File: Paper.tex [hunk 2]
```diff
@@ -1124,7 +1124,7 @@ \subsection{State \& Nonce Validation}\label{sec:statenoncevalidation}
 \end{cases}
 \end{equation}
 
-Here, $\mathtt{\small TRIE}(L_S(\boldsymbol{\sigma}_i))$ means the hash of the root node of a trie of state $\boldsymbol{\sigma}_i$; it is assumed that implementations will store this in the state database, trivial and efficient since the trie is by nature an immutable data structure.
+Here, $\mathtt{\small TRIE}(L_S(\boldsymbol{\sigma}_i))$ means the hash of the root node of a trie of state $\boldsymbol{\sigma}_i$; it is assumed that implementations will store this in the state database, trivial and efficient since the trie is by nature an immutable data structure.\hypertarget{Phi}{Phi}
```
Comment (pirapira) [inline Paper.tex]: Why isn't the target in the next paragraph?

File: Paper.tex [hunk 3]
```diff
@@ -1160,7 +1160,7 @@ \subsection{State \& Nonce Validation}\label{sec:statenoncevalidation}
 We define $\mathbf{R}[n]_{s'}$ in a similar manner.
 \begin{equation}
 \mathbf{R}[n]_{s'} =
-\Upsilon^{s}(\boldsymbol{\sigma}[n - 1], B_\mathbf{T}[n])
+\Upsilon^{s}(\boldsymbol{\sigma}[n - 1], B_\mathbf{T}[n])\hypertarget{Pi}{Pi}
```
Comment (pirapira) [inline Paper.tex]: Why isn't the target in the next paragraph, where \Pi is defined?

File: Paper.tex (hunk 1 above)
Comment (jamesray1) [reply Paper.tex]: I replied to the same question in another PR.

**Comment by pirapira** [comment]
Please use the newly defined `\linkdest` macro and move each target to the next line.

File: Paper.tex [hunk 4]
```diff
@@ -1093,7 +1093,7 @@ \subsection{Transaction Validation}
 
 \subsection{Reward Application}
 
-The application of rewards to a block involves raising the balance of the accounts of the beneficiary address of the block and each ommer by a certain amount. We raise the block's beneficiary account by $R_b$; for each ommer, we raise the block's beneficiary by an additional $\frac{1}{32}$ of the block reward and the beneficiary of the ommer gets rewarded depending on the block number. Formally we define the function $\Omega$:
+The application of rewards to a block involves raising the balance of the accounts of the beneficiary address of the block and each ommer by a certain amount. We raise the block's beneficiary account by $R_b$; for each ommer, we raise the block's beneficiary by an additional $\frac{1}{32}$ of the block reward and the beneficiary of the ommer gets rewarded depending on the block number.\hypertarget{Omega}{Omega} Formally we define the function $\Omega$:
```
Comment (pirapira) [inline Paper.tex]: Should be `\hypertarget{Omega}{}`.

File: Paper.tex [hunk 5]
```diff
@@ -1109,14 +1109,14 @@ \subsection{Reward Application}
 
 If there are collisions of the beneficiary addresses between ommers and the block (i.e. two ommers with the same beneficiary address or an ommer with the same beneficiary address as the present block), additions are applied cumulatively.
 
-We define the block reward as 3 Ether:
+\hypertarget{block-reward-R-b}{block-reward-R-b}We define the block reward as 3 Ether:
```
Comment (pirapira) [inline Paper.tex]: Similarly, the second argument should be empty.

File: Paper.tex [hunk 6]
```diff
@@ -1109,14 +1109,14 @@ \subsection{Reward Application}
...
 \begin{equation}
 \text{Let} \quad R_b = 3 \times 10^{18}
 \end{equation}
 
 \subsection{State \& Nonce Validation}\label{sec:statenoncevalidation}
 
-We may now define the function, $\Gamma$, that maps a block $B$ to its initiation state:
+\hypertarget{Gamma}{Gamma}We may now define the function, $\Gamma$, that maps a block $B$ to its initiation state:
```
Comment (pirapira) [inline Paper.tex]: Also here, the second argument should be empty.

File: Paper.tex [hunk 7]
```diff
@@ -1126,7 +1126,7 @@ \subsection{State \& Nonce Validation}\label{sec:statenoncevalidation}
 
 Here, $\mathtt{\small TRIE}(L_S(\boldsymbol{\sigma}_i))$ means the hash of the root node of a trie of state $\boldsymbol{\sigma}_i$; it is assumed that implementations will store this in the state database, trivial and efficient since the trie is by nature an immutable data structure.
 
-And finally define $\Phi$, the block transition function, which maps an incomplete block $B$ to a complete block $B'$:
+\hypertarget{Phi}{Phi}And finally define $\Phi$, the block transition function, which maps an incomplete block $B$ to a complete block $B'$:
```
Comment (pirapira) [inline Paper.tex]: Here, too.

File: Paper.tex [hunk 8]
```diff
@@ -1163,7 +1163,7 @@ \subsection{State \& Nonce Validation}\label{sec:statenoncevalidation}
 \Upsilon^{s}(\boldsymbol{\sigma}[n - 1], B_\mathbf{T}[n])
 \end{equation}
 
-Finally, we define $\Pi$ as the new state given the block reward function $\Omega$ applied to the final transaction's resultant state, $\ell(\boldsymbol{\sigma})$:
+\hypertarget{Pi}{Pi}Finally, we define $\Pi$ as the new state given the block reward function $\Omega$ applied to the final transaction's resultant state, $\ell(\boldsymbol{\sigma})$:
```
Comment (pirapira) [inline Paper.tex]: And here, too.

**Pull request merged**